content/concepts/introduction/protocols/overview.md

```diff
@@ -1,206 +1,110 @@
 ---
-title: "Overview"
-description: There are protocols everywhere you look when you're writing network applications, and libp2p is especially thick with them.
+title: "What is a libp2p Protocol"
```

Comment:
```suggestion
title: "What is a libp2p Protocol?"
```
should this have a `?`

Reply:
nope

Reply:
We usually try to leave out the quotation marks in the sidebar as it is cleaner, but they are present with in other headings and subheadings.

However, I've been meaning to get around to this to add logic for us to define the sidebar title separate from the title (like in other generators). I don't believe you can do this in Hugo by default.